### DISCUSSION TITLE
My modpack won't be approved!

### DISCUSSION THREAD

**Discussion by MeContoday** [modrinth/.github#226] [Other discussions]
I was sad that the moderator didn't leave me any message. It's been a week. My modpack is modrinth stipulation compliance. My modpack is Modrinth-compliance It's always Under Review.  What's going on here?
https://modrinth.com/modpack/newvanilla/ 
<img width="959" alt="image" src="https://github.com/modrinth/.github/assets/154036841/7b4723fa-f884-44e3-8e3f-dc6db9bc095d">
<img width="959" alt="image" src="https://github.com/modrinth/.github/assets/154036841/56a4ed47-ce69-4f1d-9c30-d9c6e0d9a32b">
<img width="959" alt="image" src="https://github.com/modrinth/.github/assets/154036841/b2766b5a-364d-480c-80e3-dada7cf25ab6">

**Comment by omo50** [comment]
there is a review delay right now. Be patient

**Reply by MeContoday** [reply]
OHHH...Yes. You are right.
